### PR TITLE
NDI-73: Fix style glitches part 3

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
# Description
This PR
- Fixes snagging list page 1 to 14
- Removes unwanted CSS
- Styles Merge duplicate contact pages

# Screenshots
**Before**
image
**After**
image

**Before**
image
**After**
image

**Before**
image
**After**
image

# Notes
Around 20 issues are fixed, major screenshots are displayed.

# Comments
- Stylelint tested
- BackstopJS tested

### CODE REVIEW

File: scss/civicrm/contact/_tabs.scss [hunk 1]
```diff
@@ -6,15 +6,16 @@
...
     padding: 0;
 
     .ui-tabs-nav {
       background: $crm-white;
       border-radius: 0;
       box-shadow: $box-shadow;
       float: left;
-      margin-bottom: -99999px;
-      padding: 0 0 99999px;
+      margin-right: 20px;
```
Comment (AkA84) [inline scss/civicrm/contact/_tabs.scss]: why did you add this right margin?

File: scss/civicrm/contact/_tabs.scss [hunk 2]
```diff
@@ -164,6 +165,7 @@
     .ui-tabs-panel {
       box-sizing: border-box;
       float: right;
+      padding: 20px 20px 0 0 !important;
```
Comment (AkA84) [inline scss/civicrm/contact/_tabs.scss]: why did you add this padding?

File: scss/civicrm/contact/pages/_contact-dedupefind.scss [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+          th {
+            border-bottom-color: $crm-grayblue-dark;
+          }
+        }
+      }
+    }  
+  }
+
+  #{civi-dialog()} {
+    min-width: 400px;
```
Comment (AkA84) [inline scss/civicrm/contact/pages/_contact-dedupefind.scss]: why do we have to customize the width of the dialog just for this page?

File: scss/civicrm/contact/pages/_contact-merge.scss [hunk 4]
```diff
@@ -0,0 +1,44 @@
+/* stylelint-disable max-nesting-depth, selector-max-compound-selectors, selector-no-qualifying-type, selector-max-id */
+
+#{civi-page('contact-merge')} {
+  .crm-container {
+    .crm-contact-merge-form-block {
+      padding-top: 20px;
+
+      .help,
+      .message.status {
+        margin: $crm-status-bar-no-top;
```
Comment (AkA84) [inline scss/civicrm/contact/pages/_contact-merge.scss]: i'm sorry to keep bringing this up every time, but the fact that we have to re-define over and over the margins for these common elements really doesn't smell right to me. Is it really possible that we can't have just *one* (or two, at best) margin value across civicrm?

File: scss/civicrm/contact/pages/_contributions.scss [hunk 5]
```diff
@@ -746,8 +746,8 @@
 
 #{civi-page('contact-view')} .ui-dialog .CRM_Contribute_Form_Contribution {
   .crm-link-credit-card-mode {
-    position: absolute;
-    right: 0;
+    float: right;
+    margin-bottom: -30px !important;
```
Comment (AkA84) [inline scss/civicrm/contact/pages/_contributions.scss]: magic number, why?

File: scss/civicrm/contact/pages/_events.scss [hunk 6]
```diff
@@ -279,6 +279,10 @@
   }
 
   .custom-group {
+    .form-layout-compressed {
+      margin-bottom: -10px;
```
Comment (AkA84) [inline scss/civicrm/contact/pages/_events.scss]: magic number, why?

File: scss/civicrm/contact/pages/_manage-groups.scss [hunk 7]
```diff
@@ -104,15 +94,16 @@
       margin-bottom: 20px;
     }
 
-    h3,
-    .accordion,
     .dataTables_wrapper {
+      @include box-shadow($crm-form-layout-shadow-lower);
+      border-radius: 0 0 $border-radius-small $border-radius-small;
```
Comment (AkA84) [inline scss/civicrm/contact/pages/_manage-groups.scss]: what's this custom border-radius? Shouldn't we have one, global?

File: scss/civicrm/contact/pages/_manage-groups.scss [hunk 8]
```diff
@@ -130,11 +121,13 @@
...
 
       .crm-accordion-body {
-        padding: 15px 0 0 !important;
-
         .section-shown {
           padding: 0;
         }
+
+        .status.no-popup {
+          margin: 0;
```
Comment (AkA84) [inline scss/civicrm/contact/pages/_manage-groups.scss]: why this custom margin?

File: scss/civicrm/contact/pages/_relationships.scss [hunk 9]
```diff
@@ -17,16 +19,17 @@
...
+      margin: 0;
       padding: 12px 20px;
     }
 
     .dataTables_wrapper {
-      border-radius: $border-radius-child;
-      box-shadow: $box-shadow;
-      margin: 20px 0;
+      @include box-shadow($crm-form-layout-shadow-lower);
+      border-radius: 0 0 $border-radius-child $border-radius-child;
```
Comment (AkA84) [inline scss/civicrm/contact/pages/_relationships.scss]: second time i see this, can't we make this generic?

File: scss/civicrm/contact/pages/_manage-groups.scss (hunk 8 above)
Comment (mukeshcompucorp) [reply scss/civicrm/contact/pages/_manage-groups.scss]: Parent is already having padding 20px so, we need to remove any default margin to this

File: scss/civicrm/contact/pages/_manage-groups.scss (hunk 7 above)
Comment (mukeshcompucorp) [reply scss/civicrm/contact/pages/_manage-groups.scss]: For both this radius comments, 
No, it depends on where data table sits, if it's looking like panel with some header, we just need border radius on bottom, or we do not need it

File: scss/civicrm/contact/pages/_contact-merge.scss (hunk 4 above)
Comment (mukeshcompucorp) [reply scss/civicrm/contact/pages/_contact-merge.scss]: We discussed this sometime ago though, based on it's parents we are not sure with multiple scenarios on where we apply what so, we came up with variables like `$crm-status-bar-no-top` and `$crm-status-bar-no-bottom`

File: scss/civicrm/contact/pages/_contact-dedupefind.scss (hunk 3 above)
Comment (mukeshcompucorp) [reply scss/civicrm/contact/pages/_contact-dedupefind.scss]: Based on the fonts and style we are using, this modal is cutting of title, so, best and easiest solution for this giving it proper height so, it doesn't look too small and have title visibly appropriate

Screenshot without width:
image

File: scss/civicrm/contact/_tabs.scss (hunk 1 above)
Comment (mukeshcompucorp) [reply scss/civicrm/contact/_tabs.scss]: when we do tab setting with page,
All childs will be aligned like table cells without space, so, to give it right space, I added 20px margin on right which is standard space we should have here.

Without margin:
image

File: scss/civicrm/contact/_tabs.scss (hunk 2 above)
Comment (mukeshcompucorp) [reply scss/civicrm/contact/_tabs.scss]: Previously we were doing it tab by tab to add padding on left and top: and hence, majority of the tabs were missing this spacing, I made it global to make it look proper.

without it:
image

File: scss/civicrm/contact/pages/_contributions.scss (hunk 5 above)
Comment (mukeshcompucorp) [reply scss/civicrm/contact/pages/_contributions.scss]: This is to make a button proper 
Before
image

After
image

File: scss/civicrm/contact/pages/_events.scss (hunk 6 above)
Comment (mukeshcompucorp) [reply scss/civicrm/contact/pages/_events.scss]: This one is to uplift tables in custom groups like below

image
